Source/FolioReaderContainer.swift

```diff
@@ -188,4 +188,11 @@ public class FolioReaderContainer: UIViewController {
     override public func preferredStatusBarStyle() -> UIStatusBarStyle {
         return isNight(.LightContent, .Default)
     }
+
+	// MARK: - Public
+
+	public func readerCenter() -> FolioReaderCenter {
```

Comment:
Why do we need this? it is already exposed thought singleton.

You can use it:
```swift
let readerCenter = FolioReader.sharedInstance.readerCenter
```